### PR TITLE
Merge develop to main

### PR DESCRIPTION
**Pull request by Bembnias**

### CODE REVIEW

File: src/components/Sections/SectionItem/SectionItem.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { TAnimal } from '@/types/animals.type'
+import Image from 'next/image'
+import styles from './SectionItem.module.scss'
+
+interface ISectionItemProps {
+  animal: TAnimal
+}
+
+const SectionItem = ({ animal }: ISectionItemProps) => {
+  const slowoIndex = animal.description.indexOf(animal.highlightedWord)
```
Comment (Sumick) [inline src/components/Sections/SectionItem/SectionItem.tsx]: **issue:** We should write code only in english

File: animals.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
+// Dummy DB to present API interactions
+
+export const animals = [
+  {
+    id: '1',
```
Comment (Sumick) [inline animals.ts]: **suggestion:** You could use UUID that you installed:)

File: src/pages/api/animals/index.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { animals } from '../../../../animals'
+import { TAnimal } from '@/types/animals.type'
+
+export default function handler(req: NextApiRequest, res: NextApiResponse<TAnimal[]>) {
```
Comment (Sumick) [inline src/pages/api/animals/index.ts:5]: **praise:** Nice one with using real API

**Review by Sumick — APPROVED** [review]
Your code looks really good. I don't have much to stick to. I will get back to you until the end of current work week.

File: src/components/Sections/SectionItem/SectionItem.tsx (hunk 1 above)
Comment (Bembnias) [reply src/components/Sections/SectionItem/SectionItem.tsx]: Thanks - fixed. 
I previously experimented with this solution outside the project and forgot to change the naming.

File: animals.ts (hunk 2 above)
Comment (Bembnias) [reply animals.ts]: That's a good point, thanks - fixed.

File: src/pages/api/animals/index.ts (hunk 3 above)
Comment (Bembnias) [reply src/pages/api/animals/index.ts:5]: Thanks, I was hoping this would be a good starting point to further scale the project 😄

**Pull request merged**